plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeModule.java

```diff
@@ -176,6 +177,12 @@ public SemiTransactionalHiveMetastore getMetastore()
                 throw new RuntimeException("SemiTransactionalHiveMetastore is not used by Delta");
             }
 
+            @Override
+            public DirectoryLister getDirectoryLister()
+            {
+                throw new RuntimeException("DirectoryLister is not used by Delta");
```

Comment:
Should this be `UnsupportedOperationException` ?

Reply:
maybe, I just made it consistent with `getMetastore` above